test/python/test_nixl_api.py

```
@@ -251,7 +251,9 @@ def test_incorrect_plugin_env(monkeypatch):
         nixl_agent("bad env agent")
 
 
-def _run_xfer_telemetry_check(agent1, agent2, expect_telemetry: bool = True) -> None:
+def _run_xfer_telemetry_check(
+    agent1, agent2, backend_name: str, expect_telemetry: bool = True
```

Ruff (0.16.0)

[warning] Boolean-typed positional argument in function definition

(FBT001)